components/map/layers/marker.md

```diff
@@ -81,20 +90,105 @@ The following example demonstrates how to configure the Map Marker Layer.
 }
 ````
 
->caption The result from the above code snippet.
+## Marker Template
+
+To customize the marker appearance, set the `Template` parameter in the corresponding marker inner tag - `MapLayerMarkerSettings`. It accepts `string` and `RenderFragment`.
```

Comment:
```suggestion
To customize the marker appearance, set the `Template` parameter in the corresponding marker inner tag - `MapLayerMarkerSettings`.
```
It doesn't work with a `RenderFragment` - see https://github.com/telerik/blazor/issues/5381#issuecomment-1438537859

> In addition, we cannot easily plug a RenderFragment (even if the Map is not written in C#) as we do with the tooltip, because the tooltip element is a single one - and we display it in an async way (on a particular user interaction), while the rendering of the markers is async - and it is preferable to avoid the N number of calls between JS and C# just to render the markers - and there can be a lot of markers. Considering other features that we can add in the future, e.g. marker clustering, such things will be even harder to be handled.

Reply:
I don't seem to have understood the comment correctly. It will be fixed.